src/web/components/Columns/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Column from '../Column';
+
+export default function Columns({ children }) {
+  return <div className="columns">{children}</div>;
+}
+
+Columns.propTypes = {
+  children: PropTypes.arrayOf(PropTypes.objectOf(Column)).isRequired,
```

Comment:
I think this should be 

```javascript
PropTypes.instanceOf(Column)
```

🤔

Reply:
When i do that i get this error : 
>    console.error node_modules/fbjs/lib/warning.js:33
      Warning: Failed prop type: Invalid prop `children[0]` of type `Object` supplied to `Columns`, expected instance of `Column`.
          in Columns

Reply:
Okay 🤔

Reply:
Maybe because it's functional component and not a class component